commonv5/ts/sonarqube/Analysis.ts

```diff
@@ -1,58 +1,26 @@
 import * as tl from "azure-pipelines-task-lib/task";
+import {
+  htmlDiv,
+  htmlLink,
+  htmlMainDiv,
+  htmlMetricList,
+  htmlMetricListItem,
+  htmlQualityGateHeader,
+  htmlSectionDiv,
+  htmlSeparator,
+} from "../helpers/html";
 import { formatMeasure } from "../helpers/measures";
-import { EndpointType } from "./Endpoint";
 import { AnalysisResult, ProjectStatus } from "./types";
 
-const qualityGateColor = (status: string) => {
-  return (
-    {
-      OK: "#6EB712",
-      WARN: "#F5B840",
-      ERROR: "#D92D20",
-    }[status] ?? "#b4b4b4"
-  );
-};
-
-const textStyle = (color = "#3E4357") =>
-  `
-  color: ${color};
-  font-family: Inter;
-  font-size: 14px;
-  font-style: normal;
-  font-weight: 500;
-  line-height: normal;
-  letter-spacing: -0.14px;`.trim();
-
-const measureStyle = (backgroundColor: string) =>
-  `background-color: ${backgroundColor};
-  padding: 4px 8px;
-  margin: 0px 4px;
-  color: #fff;
-  letter-spacing: -0.12px;
-  line-height: normal;
-  font-weight: 500;
-  font-size: 12px;`.trim();
-
-const separatorStyle = `
-  background: #EBECF0;
-  height: 1px;
-  margin: 16px 0px;`.trim();
-
 export default class Analysis {
```

Comment:
I think we could rename this class, as it is too broad considering these changes, wdyt?
Something like HTMLAnalysisReport would be a better fit imo

Reply:
I 💯% agree, and I think this class doesn't belong to the `sonarqube` folder because it's used for both `SonarQube` and `SonarCloud` - I'll rename it to your suggestion and see if I can move it out of this folder, unless you have concerns